python/pyspark/sql/utils.py

```diff
@@ -18,17 +18,24 @@
 import os
 from typing import Any, Callable, Optional, Sequence, TYPE_CHECKING, cast, TypeVar
 
-import py4j
 from py4j.java_collections import JavaArray
 from py4j.java_gateway import (
     JavaClass,
     JavaGateway,
     JavaObject,
-    is_instance_of,
 )
-from py4j.protocol import Py4JJavaError
 
 from pyspark import SparkContext
+from pyspark.errors import (  # noqa: F401
```

Comment:
Can you add a comment that these import exists for backward compatibility

Reply:
Sure, let me add a comment